pkg/apis/directpv.min.io/types/label.go

```diff
@@ -96,6 +96,40 @@ const (
 	PluginVersionLabelKey LabelKey = consts.GroupName + "/plugin-version"
 )
 
+var reservedLabelKeys = []LabelKey{
```

Comment:
declare `var reservedLabelKeys = map[LabelKey]struct{}{ ... }` and use it like `_, found := reservedLabelKeys[key]; found || strings.HasPrefix(string(key), VolumeClaimIDLabelKeyPrefix)`